.github/workflows/benchmark_nightly_gpu.yml

```diff
@@ -43,3 +43,18 @@ jobs:
         with:
           name: nightly gpu artifact
           path: /tmp/ts_benchmark
+      - name: Download benchmark artifacts for auto validation
```

Comment:
Do you mind taking a look at https://docs.github.com/en/actions/using-workflows/about-workflows to reduce some of the code duplication, it's something that's been overdue for a while and I feel it might be a good time to address

Reply:
I see they have a section on reusable workflows, but that doesn't seem to be applicable here?